packages/sql-graphql/test/inserted_updated_at.test.js

```diff
@@ -36,6 +36,11 @@ test('inserted_at updated_at happy path', async ({ pass, teardown, same, equal,
   const app = fastify()
   app.register(sqlMapper, {
     ...connInfo,
+    autoTimestamp: {
+      enabled: true,
+      createdAt: 'inserted_at',
+      updatedAt: 'updated_at'
+    },
```

Comment:
Can you please duplicate this test without any `autoTimestamp` configuration? Everything should work. Note that you'd need to change the column names to `created_at` (the new default).